### ISSUE TITLE
NOM vs OM on data portal

### ISSUE THREAD

**Issue by mslarae13** [microbiomedata/nmdc-server#885]
Some spots on the data portal say OM or organic matter and others say NOM or natural organic matter.
This should be consistent

<img width="447" alt="Screen Shot 2023-03-06 at 8 05 38 PM" src="https://user-images.githubusercontent.com/44581429/223318336-840ed64b-dadb-4813-bfdc-5464279b1cee.png">
<img width="354" alt="Screen Shot 2023-03-06 at 8 05 46 PM" src="https://user-images.githubusercontent.com/44581429/223318344-58e4fa32-6706-4aff-9d26-dea9b40f6c73.png">

Criteria for completion
- [ ] Decide on NOM or OM
- [ ] Update data portal to reflect that

**Comment by ssarrafan** [comment]
This was discussed before back in 2021. See note at https://docs.google.com/document/d/1fuJ3M689qLZIDBOJPksXgm0nvSqzacA7Y3FnEbtAt0Q/edit#bookmark=id.jewbph7fj4v2

**Comment by mslarae13** [comment]
Looks like NOM was the decision. Can we get this updated? All OM or Organic matter should be Natural Organic Matter or N. Organic Matter or NOM

@naglepuff  what would need to happen to accomplish this?

**Comment by naglepuff** [comment]
I think there are just two spots where this needs to be changed:
https://github.com/microbiomedata/nmdc-server/blob/3d88b03e3e3740128540ebc04ab602a1f8e7c2a5/web/src/encoding.ts#L349

https://github.com/microbiomedata/nmdc-server/blob/3d88b03e3e3740128540ebc04ab602a1f8e7c2a5/web/src/components/SampleListExpansion.vue#L17

**Comment by ssarrafan** [comment]
@naglepuff was this done as part of the ingests this week?

**Comment by naglepuff** [comment]
No, this isn't in dev yet. I can speed this along and get it merged in early next week.
